CHANGELOG.md

```diff
@@ -1,21 +1,21 @@
-## Unreleased
+## 2.9.4 (2024-01-24)
 
 ### All Changes
 
 #### Loki
 
 ##### Fixes
 
-* [11606](https://github.com/grafana/loki/pull/11606) **dannykopping** Fixed regression adding newlines to HTTP error response bodies which may break client integrations.
```

Comment:
Curious why this was removed?

Reply:
it didn't have the backport to 2.9.x tag, i deduced it wasn't backported? 
Let me double check if it's on the branch.

Reply:
yep, it's not backported. would you like to include it in the patch release?

Reply:
My bad! I _think_ this bug hadn't made it into 2.9.x which is why I didn't backport it
Lemme double check (and 2.8.x as well)

Reply:
OK nvm, I was right, this bug never made it into the other releases 👍 please ignore